setup.py

```diff
@@ -110,7 +110,7 @@ def _config_exe(exe_name):
     author_email="bbp-ou-hpc@groupes.epfl.ch",
     description="NEURON Modeling Language Source-to-Source Compiler Framework",
     long_description="",
-    packages=["nmodl"],
+    packages=["python"],
```

Comment:
Question : should we really change package name in setup.py? does this used somewhere else? e.g. pip show or pip list command?